### ISSUE TITLE
16 in SineSubtract.cxx might need to be removed

### ISSUE THREAD

**Issue by MyoungchulK** [nichol77/libRootFftwWrapper#5]
[16](https://github.com/nichol77/libRootFftwWrapper/blob/bb945e6b6d5afc82c75b6cb21dc8d1ceb17446c0/src/SineSubtract.cxx#L1154) is altering the magnitude of `power_spectra` values. 
And `SineSubtract.cxx` is using the same `power_spectra` as an [initial guess](https://github.com/nichol77/libRootFftwWrapper/blob/bb945e6b6d5afc82c75b6cb21dc8d1ceb17446c0/src/SineSubtract.cxx#L1239) of fit.
Since several [Set~LimitedVariable](https://github.com/nichol77/libRootFftwWrapper/blob/bb945e6b6d5afc82c75b6cb21dc8d1ceb17446c0/src/SineSubtract.cxx#L787) functions (line 787 ~ 803) in the [dofit](https://github.com/nichol77/libRootFftwWrapper/blob/bb945e6b6d5afc82c75b6cb21dc8d1ceb17446c0/src/SineSubtract.cxx#L722) are using [amp](https://github.com/nichol77/libRootFftwWrapper/blob/bb945e6b6d5afc82c75b6cb21dc8d1ceb17446c0/src/SineSubtract.cxx#L186) (`power_spectra`) values on the first attempt of [Minimize()](https://github.com/nichol77/libRootFftwWrapper/blob/bb945e6b6d5afc82c75b6cb21dc8d1ceb17446c0/src/SineSubtract.cxx#L818), It might give wrong minimization results to users
Hope This makes sense!

**Comment by cozzyd** [comment]
Yes, this appears to be vestigial somehow. Can you check if removing it doesn't somehow blow things up?
